frontend/src/components/TableUnratedTaskRow.tsx

```diff
@@ -72,12 +72,28 @@ export const TableUnratedTaskRow: React.FC<TableTaskRowProps> = ({ task }) => {
   */
   useEffect(() => {
     if (type === RoleType.Admin) {
-      const ratingIds = task.ratings.map((rating) => rating.createdByUser);
+      const ratings = task.ratings.map((rating) => {
+        return {
+          createdByUser: rating.createdByUser,
+          forCustomer: rating.forCustomer,
+        };
```

Comment:
This can be written without the return statement

```
const ratings = task.ratings.map((rating) => ({
  createdByUser: rating.createdByUser,
  forCustomer: rating.forCustomer,
}));
```